File: AndroidStudioProjects/listviewjava/app/src/main/java/com/example/listviewjava/MainActivity.java
package com.example.listviewjava;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.ListView;

import java.lang.reflect.Array;

public class MainActivity extends AppCompatActivity {
    String [] name = new String[]
            {
                    "A","Q" , "E" , "T", "Y"
            };

ListView l1;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
 l1 = findViewById(R.id.lsv);

        ArrayAdapter<String> NAME= new ArrayAdapter<>(
                this,
                android.R.layout.simple_list_item_1,
                android.R.id.text1,
            name

        );

        l1.setAdapter(NAME);




    }
}

File: AndroidStudioProjects/listview/app/src/main/java/com/example/listview/MainActivity.kt
package com.example.listview

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ArrayAdapter
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.fruits.*
import kotlinx.android.synthetic.main.fruits.view.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        lvf.adapter = ArrayAdapter<String>(
            this,
            R.layout.fruits,
            R.id.textv ,
                arrayOf("apple" , "mango" , "a" , "b" , "c", "d" , "e" , "f" , "g")

        )

        lvf.setOnItemClickListener { parent, view, position, id ->
            Toast.makeText(this , "shivek ate ${position+1} ${view.textv.text}" ,Toast.LENGTH_LONG).show()

        }
    }
}
